internal/pkg/build/sources/conveyorPacker_buildkit.go

```diff
@@ -0,0 +1,352 @@
+// Copyright (c) Contributors to the Apptainer project, established as
+//   Apptainer a Series of LF Projects LLC.
+//   For website terms of use, trademark policy, privacy policy and other
+//   project policies see https://lfprojects.org/policies
+// Copyright (c) 2020, Control Command Inc. All rights reserved.
+// Copyright (c) 2018-2024, Sylabs Inc. All rights reserved.
```

Comment:
Is there anything significant in this file that was copied from other files, or is it essentially from scratch?

Reply:
It is a copy of (big) parts of "net", and some functions (from it) are called directly as well...

It would be possible to refactor those into common functions, but since they were hardcoded in "net"